apps/settings/js/bluetooth.js

```diff
@@ -664,13 +666,14 @@ navigator.mozL10n.once(function bluetoothSettings() {
         }
       } else {
         // show pair process fail.
-        var msg = _('error-pair-title');
+        var l10nId = 'error-pair-title';
         if (errorMessage === 'Repeated Attempts') {
-          msg = msg + '\n' + _('error-pair-toofast');
+          l10nId = 'error-pair-toofast';
         } else if (errorMessage === 'Authentication Failed') {
-          msg = msg + '\n' + _('error-pair-pincode');
+          l10nId = 'error-pair-pincode';
```

Comment:
Hmm, I actually think that the UI would be less weird :)

Would you be ok if I took some screenshots with this change (and without) and NI'ed UX in the bug?